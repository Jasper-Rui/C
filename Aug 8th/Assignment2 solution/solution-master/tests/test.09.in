TRX 11235 Alice_Q_Public Bob_X_Private 150000 20
BLK 1 0 0x00000000 1
11235 Alice_Q_Public Bob_X_Private 150000 20
0x00000065 0x006a00e3
MINE
END

