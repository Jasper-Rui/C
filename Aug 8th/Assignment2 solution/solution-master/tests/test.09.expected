Adding transaction: 11235 Alice_Q_Public Bob_X_Private 150000 20
Removing transaction: 11235 Alice_Q_Public Bob_X_Private 150000 20
Block mined: 2 1 0x006a00e3 0
0x00000017 0x00ddb1cb
